integration-tests/ccip-tests/actions/ccip_helpers.go

```diff
@@ -1750,7 +1751,12 @@ func (sourceCCIP *SourceCCIPModule) CCIPMsg(
 		return router.ClientEVM2AnyMessage{}, fmt.Errorf("failed encoding the receiver address: %w", err)
 	}
 
-	extraArgsV1, err := testhelpers.GetEVMExtraArgsV1(gasLimit, false)
+	var extraArgs []byte
+	if sourceCCIP.OnRamp.Instance.V1_2_0 != nil {
+		extraArgs, err = testhelpers.GetEVMExtraArgsV1(gasLimit, allowOutOfOrder)
```

Comment:
I think we can keep that as false to avoid confusion.
```suggestion
		extraArgs, err = testhelpers.GetEVMExtraArgsV1(gasLimit, false)
```